micropython/bundles/README.md

```diff
@@ -0,0 +1,6 @@
+These are "meta packages" designed to make it easy to ensure that boards end
+up with a consistent set of default packages.
```

Comment:
This sentence (and the `metadata` in the `manifest.py`) are all very board specific.  Does it make sense to reword it to be a bit more generic, eg "to ensure that deployments"?

Reply:
Updated